easybuild/easyconfigs/l/LAMMPS/LAMMPS-24Oct2018-intel-2018b.eb

```diff
@@ -0,0 +1,143 @@
+easyblock = 'CMakeMake'
+
+name = 'LAMMPS'
+version = '24Oct2018'
+patch_release = True
+
+# if you turn this on you should tweak your hardware in the kokkos options below
+kokkos = False
+
+# CUDA switch, if used will need to tweak hardware below
+cuda = False
+
+versionsuffix = ''
+if kokkos:
+    versionsuffix += '-kokkos'
+if cuda:
+    versionsuffix += '-CUDA'
+
+homepage = 'http://lammps.sandia.gov/'
+description = """LAMMPS is a classical molecular dynamics code, and an acronym
+for Large-scale Atomic/Molecular Massively Parallel Simulator. LAMMPS has
+potentials for solid-state materials (metals, semiconductors) and soft matter
+(biomolecules, polymers) and coarse-grained or mesoscopic systems. It can be
+used to model atoms or, more generically, as a parallel particle simulator at
+the atomic, meso, or continuum scale. LAMMPS runs on single processors or in
+parallel using message-passing techniques and a spatial-decomposition of the
+simulation domain. The code is designed to be easy to modify or extend with new
+functionality.
+"""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+toolchainopts = {'openmp': True, 'cstd': 'c++11'}
+
+source_urls = ['https://github.com/lammps/lammps/archive/']
+if patch_release:
+    sources = ['patch_%(version)s.tar.gz']
+    source_dir_name = '%(namelower)s-patch_%(version)s'
+else:
+    sources = ['%(version)s.tar.gz']
+    source_dir_name = '%(namelower)s-%(version)s'
+
+checksums = ['a171dff5aff7aaa2c9606ab2abc9260f2b685a5c7f6d650e7f2b59cf4aa149d6']
+
+builddependencies = [
+    ('CMake', '3.12.1'),
+    ('pkg-config', '0.29.2')
+]
+
+dependencies = [
+    ('Python', '2.7.15'),
+    ('libpng', '1.6.34'),
+    ('libjpeg-turbo', '2.0.0'),
+    ('netCDF', '4.6.1'),
+    ('GSL', '2.5'),
+    ('zlib', '1.2.11'),
+    ('gzip', '1.9'),
+    ('HDF5', '1.10.2'),
+    ('tbb', '2018_U5'),
+    #('CUDA', '10.0.130', '', True),
```

Comment:
It would seem that CUDA+kokkos support only works with GCC, the wrapper causes problems in other modules using (CUDA supported) Intel compilers.